src/components/pageTemplates/NodeTprUnitPage.tsx

```diff
@@ -113,23 +158,47 @@ function NodeTprUnitPage({ node, sidebar, ...props }: NodeTprUnitProps): JSX.Ele
             )}
 
             <ContactInfo />
-            
+
             {field_content?.length > 0 && (
-              <ContentMapper content={field_content} pageType='tpr_unit' locationId={drupal_internal__id} mapId={service_map_embed} />
+              <ContentMapper
+                content={field_content}
+                pageType='tpr_unit'
+                locationId={drupal_internal__id}
+                mapId={service_map_embed}
+              />
             )}
           </div>
-          <div className="sidebar col col-4 flex-order-first">
-            <Sidebar {...sidebar}/>
+          <div className='sidebar col col-4 flex-order-first'>
+            <Sidebar {...sidebar} />
             <ContactInfo aside={true} />
           </div>
         </div>
-        <div className="columns">
-          <div className="lower-content-region col col-12">
+        <div className='columns'>
+          <div className='lower-content-region col col-12'>
             {field_lower_content?.length > 0 && (
-              <ContentMapper content={field_lower_content} pageType='tpr_unit' />
+              <ContentMapper
+                content={field_lower_content}
+                pageType='tpr_unit'
+              />
             )}
           </div>
         </div>
+        {node.accessibility_sentences.length !== 0 && (
+          <AccordionWithIcon
+            ariaLabel={t('unit.accessibility_information')}
+            accordionTitle={t('unit.accessibility_information')}
+            data={node.accessibility_sentences}
```

Comment:
This should be refactored so the data received by the accordion component is preformatted. Now it depends on the group formatting and the accessibility sentences data, and might break if we use the component for something else.